src/subdevice_group/mod.rs

```diff
@@ -854,6 +869,11 @@ where
         self.process_pdi_response(&data)
     }
 
+    /// TODO
+    pub fn pdi_range(&self) -> Range<u32> {
```

Comment:
In my case I'm actually doing some manual interaction with the PDI, so it was helpful to know the entire range for the group.